src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -179,6 +168,65 @@ public void shutdown() {
         }
     }
 
+    public class CertificateAuthorityChangeHandler {
+
+        private BatchedSubscriber subscriber;
+
+        @SuppressWarnings({"unchecked", "PMD.UnusedFormalParameter"})
+        private void onCAChange(WhatHappened what, Set<Node> whatChanged) {
+            Topic caTopic;
+            try {
+                caTopic = findCATopic();
+            } catch (ServiceLoadException e) {
+                serviceErrored(e);
+                return;
+            }
+
+            List<String> caCerts = (List<String>) caTopic.toPOJO();
```

Comment:
don't use topojo here and definitely don't blindly cast.

Use Coerce.toStringList

Reply:
whoops, yeah will adjust